politeiawww/api/v1/api.md

```diff
@@ -872,7 +850,23 @@ SHALL observe.
 
 **Params:** none
 
-**Results:** see below
+**Results:** 
+
+| | Type | Description |
+|-|-|-|
+| passwordminchars | integer | minimum number of characters accepted for user passwords |
+| proposallistpagesize | integer | maximum number of proposals returned for the routes that return lists of proposals |
+| maximages | integer | maximum number of images accepted when creating a new proposal |
+| maximagesize | integer | maximum image file size (in bytes) accepted when creating a new proposal |
+| maxmds | integer | maximum number of markdown files accepted when creating a new proposal |
+| maxmdsize | integer | maximum markdown file size (in bytes) accepted when creating a new proposal |
+| validmimetypes | array of strings | list of all acceptable MIME types that can be communicated between client and server. |
+| maxnamelength | integer | max length of a proposal name |
```

Comment:
Same comment as above, this was renamed to `minproposalnamelength`, `maxproposalnamelength`, and `proposalnamesupportedchars`.